pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/model/elastic/migration/OsteoMigratorTest.java

```diff
@@ -171,15 +172,15 @@ public void testOsteoMigrator() {
         verifyExportLogs(exportLogs);
 
         // should be onc history for OS1 only participant
-        String os1DdpPtpId = os1Ptp.getDdpParticipantIdOrThrow();
-        verifyOncHistory(os1DdpPtpId, List.of(os1Ptp.getParticipantIdOrThrow()));
+        String os1DdpPtpId = os1Ptp.getRequiredDdpParticipantId();
```

Comment:
Opportunistically changing to the preferred method name `getRequiredDdpParticipantId`.